src/microbe_stage/AgentProjectile.cs

```diff
@@ -35,12 +46,30 @@ public void OnBodyEntered(Node body)
             }
         }
 
+
+        DespawnTimer.Start();
+    }
+
+    public void ApplyPropertiesFromSave(AgentProjectile projectile)
```

Comment:
This is the approach used by the old version of the save system. I'm quite sure that this method (at least with current master branch version), would never get called at all...